build.sbt

```diff
@@ -272,6 +272,15 @@ lazy val logstageDi = inLogStage.as.module
     distageCore
   ).map(_.testOnlyRef))
 
+lazy val logstageConfig = inLogStage.as.module
+  .depends(fundamentalsTypesafeConfig, logstageApiLogger, logstageRenderingCirce)
```

Comment:
logstageRenderingCirce is never used inside that module, you may drop the dependency